csp/utils.py

```diff
@@ -117,17 +117,20 @@ def build_policy(
 
     for key, value in csp.items():
         # Check for boolean directives.
-        if len(value) == 1 and isinstance(value[0], bool):
-            if value[0] is True:
-                policy_parts[key] = ""
-            continue
+        if len(value) == 1:
+            val = list(value)[0]
+            if isinstance(val, bool):
+                if value[0] is True:
+                    policy_parts[key] = ""
+                continue
```

Comment:
I had to think about this a minute. This appears to allow `{True}`, `[True]` and `(True,)`, although a plain `True` is the expected value, wrapped in a sequence on line 108.